apps/site/assets/ts/stop/components/DepartureCard.tsx

```diff
@@ -34,28 +36,26 @@ const DepartureCard = ({
         {routeName(route)}
       </a>
       {/* TODO can we avoid hard coding the direction ids? */}
-      {!departuresByDirection[0] && !departuresByDirection[1] ? (
-        <div className="departure-card__headsign-name">No upcoming trips</div>
-      ) : (
-        <>
-          <DepartureTimes
-            key={`${route.id}-0`}
-            route={route}
-            directionId={0}
-            departuresForDirection={departuresByDirection[0]}
-            onClick={onClick}
-            alertsForDirection={allAlertsForDirection(alertsForRoute, 0)}
-          />
-          <DepartureTimes
-            key={`${route.id}-1`}
-            route={route}
-            directionId={1}
-            departuresForDirection={departuresByDirection[1]}
-            onClick={onClick}
-            alertsForDirection={allAlertsForDirection(alertsForRoute, 1)}
-          />
-        </>
-      )}
+      <>
```

Comment:
suggestion: The logic in `<DepartureTimes>` is growing quite complex and I wonder if it makes sense to make this problem `<DepartureCard>`'s responsibility instead. Maybe something like 
```
{
  departuresByDirection[0].length > 0 
  ? <DepartureTimes {/* existing stuff */} /> 
  : !isAtDestination(stopId, route, 0) && <> 
      <div className="departure-card__headsign-name">{route.direction_destinations[0]}</div>
      <div>No upcoming trips</div>
    </>
}
```